samples/subsys/logging/syst/prj.conf

```diff
@@ -18,3 +18,4 @@ CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=100
 CONFIG_LOG_PROCESS_TRIGGER_THRESHOLD=2
 CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=256
 CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=4096
+CONFIG_LOG2_MODE_DEFERRED=y
```

Comment:
You are changing this test to do v2 logging, and v1 is no longer being tested.

Reply:
Yeahh you are right. Removed the the v2 logging config from prj.conf and added support for testing sys-t sample with v2 logging subsystem as well in sample.yaml. Thanks!!